README.md

```diff
@@ -1,3 +1,128 @@
-# Cartoomics-Grant
-![Screen Shot 2022-08-19 at 10 34 46 AM](https://user-images.githubusercontent.com/70932395/185665797-f3f3ccc0-c070-45bb-ae7e-e6d8acf5b664.png)
-<img width="824" alt="Screen Shot 2022-08-19 at 9 43 43 AM" src="https://user-images.githubusercontent.com/70932395/185656682-fff53d5c-eecf-4b0d-a259-2fa260385ba5.png">
+# Cartoomics Figure Generation 
+
+This Cartoomics repository consists of a workflow written in python that can generate an interaction network using a given knowledge graph from a user derived description of a cartoon image. The goal of this workflow is to generate complete and detailed pathway diagrams using the vast information contained within a knowledge graph.
+
+## Getting Started
+
+These instructions will provide the necessary environments, programs with installation instructions, and input files in order to run the creating_subgraph_using_cosinesim.py script, which will run this workflow from end to end.
+
+The path search algorithms available are:
+
+- Shortest Path Search: find the shortest path between each source and target node (may not be identical each time)
+- Cosine similarity Prioritization: prioritize all shortest paths between each source and target node by maximizing total cosine similarity of the start and all intermediate nodes to the target node in each path
+- Path Degree Product Prioritization: prioritize all shortest paths between each source and target node by maximizing path degree product of all nodes in each path
+
+This ReadMe currently describes the Cosine similarity Prioritization algorithm.
+
+### Prerequisites
+The following software or software packages must be installed, as detailed in the requirements.txt:
```

Comment:
Do we need to add basic libraries, like pandas? Is there an automated way to produce this list?

Reply:
I wasn't sure, I tried to keep it broad for now. But potentially just pull from the requirements.txt